lib/eig_trlm_3d.cpp

```diff
@@ -0,0 +1,701 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+#include <quda_internal.h>
```

Comment:
Should we start implementing using `#include "..."` for headers inside QUDA, `#include <...>` for external headers with this PR? Or should we just do a mega-change at some point in the future?